src/symbol/sprite_atlas.js

```diff
@@ -205,30 +251,26 @@ class SpriteAtlas extends Evented {
         this.dirty = true;
     }
 
-    setSprite(sprite) {
-        if (sprite && this.canvas) {
-            this.canvas.width = this.width;
-            this.canvas.height = this.height;
-        }
```

Comment:
Yeah, `this.canvas` is never set. I think it was related to some debug code that got removed.